package.json

```diff
@@ -41,23 +41,23 @@
     "postinstall": "patch-package",
     "prebuild": "rimraf build/ build-bin/",
     "build": "npm run build:src && npm run catch-uncommitted",
-    "build:src": "npm run prettify && npm run lint && npm run build:automation && npm run build:fast && npm run build:doc",
+    "build:src": "npm run prettify && npm run lint && npm run build:fast && npm run build:test && npm run build:doc",
     "build:fast": "gulp build && tsc",
-    "build:automation": "tsc -P automation/tsconfig.json --noEmit",
-    "build:doc": "mkdirp doc/ && ts-node --transpile-only -P automation/tsconfig.json automation/capitanodoc/index.ts > doc/cli.markdown",
-    "build:standalone": "ts-node --transpile-only -P automation/tsconfig.json automation/run.ts build:standalone",
-    "build:installer": "ts-node --transpile-only -P automation/tsconfig.json automation/run.ts build:installer",
+    "build:test": "tsc -P ./tsconfig.dev.json --noEmit",
+    "build:doc": "mkdirp doc/ && ts-node --transpile-only automation/capitanodoc/index.ts > doc/cli.markdown",
```

Comment:
Nice to get rid of the `-P` argument! 👍

Reply:
Yeah I realized it doesn't actually do anything when using `--transpile-only` so it's nicer to have it cleaned up